packages/core/utils/dependencies.ts

```diff
@@ -30,7 +30,6 @@ export async function suggestInstallingDependencies(workingDirectory: string) {
     }
 
     if (!selectedPm || !packageManagers[selectedPm]) {
-        console.log("Skipped installing dependencies");
```

Comment:
Removed this log for two reasons: 
- it wasn't using clack's logger so the alignment was off
- it's also redundant

The latter reason is really the core reason to remove it
![img](https://i.imgur.com/IqNJJm8.png)
the prompt with the response already conveys that we don't want to install deps, so there's no need to say that we're _not_ doing something

Reply:
Great catch!